src/main/java/approximations/java/lang/StringBuffer_Tests.java

```diff
@@ -0,0 +1,2150 @@
+package approximations.java.lang;
+
+
+import approximations.java.Test;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public final class StringBuffer_Tests {
+
+    private static final int MAX_CODE_POINT = 1114111;
+    private static final int MIN_CODE_POINT = 0;
+
+    // internal variables
+
+    // constructors
+
+    // StringBufferAutomaton::StringBuffer
+    @Test(executionMax = 0)
```

Comment:
If `executionMax` is 0 (zero) then annotation is unnecessary. It will be guessed automatically. This is the default behavior.